Scripts/Modules/TooltipModule/TooltipModule.cs

```diff
@@ -177,14 +188,27 @@ void UpdateVisuals(ITooltip tooltip, TooltipUI tooltipUI, Transform target, floa
 
 		public void OnRayEntered(GameObject gameObject, RayEventData eventData)
 		{
+			if (gameObject == this.gameObject)
```

Comment:
This was a fix for #256. I opened another issue (#278) for the general case, since it came back up again.